cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -837,6 +837,56 @@ def _generate_head_node_pass_role_resources(self):
         return list(pass_role_resources)
 
 
+class LoginNodesIamResources(NodeIamResourcesBase):
+    """Construct defining IAM resources for a login node."""
+
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
+        config: BaseClusterConfig,
+        node: Union[HeadNode, BaseQueue, LoginNodesPool],
+        shared_storage_infos: dict,
+        name: str,
+    ):
+        super().__init__(scope, id, config, node, shared_storage_infos, name)
+
+    def _build_policy(self) -> List[iam.PolicyStatement]:
+        return [
+            iam.PolicyStatement(
+                actions=[
+                    "cloudwatch:PutMetricData",
+                    "ec2:DescribeVolumes",
+                    "ec2:DescribeTags",
+                    "logs:PutLogEvents",
+                    "logs:DescribeLogStreams",
+                    "logs:DescribeLogGroups",
+                    "logs:CreateLogStream",
+                    "logs:CreateLogGroup",
+                ],
+                effect=iam.Effect.ALLOW,
+                resources=["*"],
+            ),
+            iam.PolicyStatement(
+                actions=["ssm:GetParameter"],
+                effect=iam.Effect.ALLOW,
+                resources=["arn:aws:ssm:*:*:parameter/AmazonCloudWatch-*"],
+            ),
```

Comment:
This seems to be a duplicated of the `CloudWatchAgentServerPolicy`: why do you need to duplicate it?